exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/Serializer.java

```diff
@@ -95,7 +95,12 @@ public void serializeInt64(ProtoFieldInfo field, long value) throws IOException
 
   /** Serializes a protobuf {@code fixed64} field. */
   public void serializeFixed64(ProtoFieldInfo field, long value) throws IOException {
-    if (value == 0) {
+    serializeFixed64(field, value, /* force= */ false);
+  }
+
+  /** Serializes a protobuf {@code fixed64} field. */
+  public void serializeFixed64(ProtoFieldInfo field, long value, boolean force) throws IOException {
```

Comment:
Better to just make `writeFixed64` public I think. If the name is confusing, it could be renamed to `serializeRequiredFixed64` or something

Reply:
Yeah, I'd like to make it public.

Reply:
I added an optional method to mirror the "size" method calls, as I think that reduces confusion the most (and simplifies reading this later).